kzg_prover/benches/kzg.rs

```diff
@@ -8,8 +8,8 @@ use summa_solvency::{
     circuits::{
         univariate_grand_sum::UnivariateGrandSum,
         utils::{
-            full_prover, generate_setup_artifacts, open_grand_sums, open_user_points,
-            open_user_points_amortized, verify_grand_sum_openings, verify_user_inclusion,
```

Comment:
If open_user_points_amortized is deleted, will it break the benchmark?

Reply:
Ah, yes. You are right. 
Will fix it.